src/app/components/cookie-policy/cookie-policy.component.ts

```diff
@@ -1,10 +1,41 @@
-import { Component } from '@angular/core';
+import {Component} from '@angular/core';
```

Comment:
_nitpick:_
Looks like you have a formatter that _removes_ the spaces but I think the linter will put them back in so this will likely end up in a tug-of-war.